### PR TITLE
(minimal) changes for OCaml 5.0.0~beta2 compatibility

### PR DESCRIPTION
**Pull request by chetmurthy**
Very minimal changes for 5.0.0 compatibility,  since AST didn't change.

### CODE REVIEW

**Comment by chetmurthy** [comment]
It looks like the CI failures were all due to some sort of CI machinery errors?  Problems with git repos, locks ?

File: packages/elpi/elpi.1.15.0/opam [hunk 1]
```diff
@@ -14,7 +14,7 @@ build: [
 ]
 
 depends: [
-  "ocaml" {>= "4.07.0" }
+  "ocaml" {>= "4.07.0" & <= "4.15.0" }
```
Comment (mseri) [inline packages/elpi/elpi.1.15.0/opam:17]: Why <= 4.15?

Comment (chetmurthy) [reply packages/elpi/elpi.1.15.0/opam:17]: Arrgh, I think "I made a mistake".  *grin*  I wanted to exclude all ocaml 5.X versions.  Let me try again.  This time, I'll use
` < "4.15.0"`.  Though I wonder if `< "5"` would work.

**Comment by chetmurthy** [comment]
Looks like more unrelated test-setup  errors.

**Comment by chetmurthy** [comment]
Hi, it looks like all the failures are unrelated to this PR; can you let me know if there's something else I need to do?

File: packages/elpi/elpi.1.15.0/opam (hunk 1 above)
Comment (mseri) [reply packages/elpi/elpi.1.15.0/opam]: This is strange, the original bound should have worked

File: packages/elpi/elpi.1.15.0/opam [hunk 2]
```diff
@@ -14,7 +14,7 @@ build: [
 ]
 
 depends: [
-  "ocaml" {>= "4.07.0" }
+  "ocaml" {>= "4.07.0" & < "4.15.0" }
```
Comment (mseri) [inline packages/elpi/elpi.1.15.0/opam:17]:
```suggestion
  "ocaml" {>= "4.07.0" & < "5" }
```

**Comment by mseri** [comment]
Thanks

**Comment by chetmurthy** [comment]
Arrgh!  I screwed up and pushed the wrong code.  This isn't actually 5.0.0 support (it's just the same as pre-5.0.0).  I'll prepare a fix; can I send a PR to remove this release ?

**Comment by chetmurthy** [comment]
Urrrgh.  No, actually, this release is fine.  Thank you for asking.  I had thought I'd screwed it up, but no, actually, my "thinking it was screwed-up" was screwed-up.  Sorry for the false alarm!
